src/deck/drivers/src/ledring12.c

```diff
@@ -138,8 +139,10 @@ typedef void (*Ledring12Effect)(uint8_t buffer[][3], bool reset);
   dest[1] = ((uint16_t)G6 * 259 + 33) >> 6;                                    \
   dest[2] = ((uint16_t)B5 * 527 + 23) >> 6;
 
-#ifndef LEDRING_DEFAULT_EFFECT
-#define LEDRING_DEFAULT_EFFECT 6
+#ifdef CONFIG_DECK_LEDRING_DEFAULT_EFFECT
```

Comment:
Same as above, the `#ifdef` should not be required and default value should in in kconfig.

Reply:
Fixed in 0afe821